src/Microsoft.ML.AutoML/Microsoft.ML.AutoML.csproj

```diff
@@ -11,10 +11,13 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.ML" Version="$(MlDotNetPackageVersion)" />
-    <PackageReference Include="Microsoft.ML.LightGBM" Version="$(MlDotNetPackageVersion)" />
-    <PackageReference Include="Microsoft.ML.Mkl.Components" Version="$(MlDotNetPackageVersion)" />
-    <PackageReference Include="Microsoft.ML.Recommender" Version="$(MlDotNetPackageVersion)" />
+    <PackageReference Include="Microsoft.ML" Version="1.4.0-preview3-28218-2" />
+    <PackageReference Include="Microsoft.ML.LightGBM" Version="1.4.0-preview3-28218-2" />
```

Comment:
Should it be `LightGbm`?

Reply:
note: LightGbm in all uppercasing is also in nupkgproj files (pkg\..AutoML and pkg\mlnet)